testsuite/docs-examples-cpp/run.py

```diff
@@ -11,7 +11,7 @@
     prefix = "./build/"
 
 # Prep:
-command += run_app("cmake -E copy ../common/grid-small.exr grid.exr")
+command += run_app("cmake -E copy ${OIIO_TESTSUITE_ROOT}/common/grid-small.exr grid.exr")
```

Comment:
Why this change?

Reply:
Related to other [note](https://github.com/AcademySoftwareFoundation/OpenImageIO/pull/4016#discussion_r1357463066)